src/test/java/net/minestom/server/instance/block/handler/BlockHandlerIntegrationTest.java

```diff
@@ -0,0 +1,123 @@
+package net.minestom.server.instance.block.handler;
+
+import net.minestom.server.coordinate.Vec;
+import net.minestom.server.entity.PlayerHand;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.instance.block.BlockFace;
+import net.minestom.server.instance.block.BlockHandler;
+import net.minestom.server.network.packet.client.play.ClientPlayerBlockPlacementPacket;
+import net.minestom.server.utils.NamespaceID;
+import net.minestom.testing.Env;
+import net.minestom.testing.EnvTest;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@EnvTest
+public class BlockHandlerIntegrationTest {
```

Comment:
Nitpick: Remove public for junit5.
Ref: https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-classes-and-methods

Reply:
I would prefer for the whole class